packages/htm/README.md

```diff
@@ -18,34 +18,36 @@ npm install @remote-ui/htm --save
 
 ## Usage
 
-`@remote-ui/htm` provides two functions. The first, `htm`, is a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which binds the main function from the `htm` library into a basic object structure. `render`, the second function exported from this library, takes this generated structure and a [`@remote-ui/core` `RemoteRoot`](../core#remoteroot), and appends all of the resulting components into that root.
+`@remote-ui/htm` provides two functions. The first, `createRender`, is a factory function that takes a [`@remote-ui/core` `RemoteRoot`](../core#remoteroot), and returns a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates). This tagged template binds the main function from the `htm` library into creating a tree of `@remote-ui/core` `RemoteComponent` and `RemoteText` objects. The second function, `append`, is a helper to call `appendChild()` on a `RemoteRoot` or `RemoteComponent` for each node in the tree returned by the tagged template.
 
 ```ts
 import {createRemoteRoot} from '@remote-ui/core';
-import {htm, render} from '@remote-ui/htm';
+import {createRender, append} from '@remote-ui/htm';
 
 const root = createRemoteRoot(() => {});
+const ui = createRender(root);
+
 const onPress = () => {};
 
-render(htm`<Button primary onPress=${onPress}>Submit</Button>`, root);
-root.mount();
```

Comment:
Is `.mount()` no longer needed?

Reply:
It is still needed, but I recently removed it from a few other READMEs since we changed Argo to implicitly call this method (not everyone will be using Argo, but I want to make the READMEs support that use case best)